### DISCUSSION TITLE
Are there planns to support inbetween classes like border-3 ?

### DISCUSSION THREAD

**Discussion by TDroogers** [tailwindlabs/tailwindcss#6664] [Ideas]
Hi, with tailwind 3.x the size of the total package is not important any more, does this mean we can expect to get more smaller steps in classes?

In example we have border-2 and border-4, but border-3 does not work, however logically, this should be {border-width: 3px;}

This is of course only one of the many examples that could allow us to fine tune a lot smaller details.

**Comment by NovusOpus** [comment]
You can always use [arbitrary values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values), example: `border-[3px]` or more custom values like `border-[3rem]` `border-[3.5rem]` .
